css/styles.css

```diff
@@ -6,10 +6,14 @@ body {
     font-family: 'Press Start 2P', cursive;
 }
 
-.container {
+#container {
     width: 1000px;
     height: 800px;
     margin: 0 auto;
     border: 1px solid black;
 
+}
+
+.ie .splash {
+    background-color: blue;
```

Comment:
should be a hex code (that matches the blue from the gradient version)

Reply:
🎉